src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -146,12 +148,17 @@ export function computeAlterationTypes(alterationData: ICaseAlteration[]): strin
 }
 
 export function getPseudoOqlSummaryByAlterationTypes(oqlData: {[oqlLine: string]: IOqlData}, oqlLine: string, alterationTypes: string[]) : PseudoOqlSummary {
-    const alteratedPseudoOqlSummarys = _.chain(alterationTypes)
-        .map(type => generatePseudoOqlSummary(oqlData, oqlLine, type))
-        .filter((summary) => summary ? summary.summaryClass === styles.alterationSpan : false)
-        .value();
+    const pseudoOqlSummarys = _.map(alterationTypes, (type) => generatePseudoOqlSummary(oqlData, oqlLine, type));
```

Comment:
plural of summary is actually "summaries."   I apologize on behalf of the English language! I think this is true of all words ending in "y".  E.g. study => studies.